hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -97,8 +97,9 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
     }
 
     newCreateBucketRequest.setBucketInfo(newBucketInfo.build());
+
     return getOmRequest().toBuilder().setUserInfo(getUserInfo())
-        .setCreateBucketRequest(newCreateBucketRequest.build()).build();
+       .setCreateBucketRequest(newCreateBucketRequest.build()).build();
   }
```

Comment:
We are not setting the new omRequest in OMClientRequest anymore?

Reply:
Yes.